pom.xml

```diff
@@ -2586,6 +2591,100 @@
         <zookeeper.version>3.4.6</zookeeper.version>
         <curator.version>2.6.0</curator.version>
       </properties>
+      <dependencyManagement>
+        <dependencies>
+          <!--
+          the AWS module pulls in jackson; its transitive dependencies can create
+          intra-jackson-module version problems.
+          -->
+          <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-aws</artifactId>
```

Comment:
these are declaring the existence of the dependencies, and their exclusions, *but not their actual use*. This is in keeping with standard the maven multiproject best practice of declaring your dependency in one place, so it's consistent in all uses across the project. Keeps maintenance costs down. 

This declaration here doesn't add anything to the CP, only that if you build with {{-Phadoop-2.7}}, those are the specific versions of the relevant hadoop-* JARs to pull in